Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/MixedRealitySpeechCommandsProfile.cs

```diff
@@ -12,6 +14,35 @@ namespace Microsoft.MixedReality.Toolkit.Core.Definitions.InputSystem
     [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Speech Commands Profile", fileName = "MixedRealitySpeechCommandsProfile", order = (int)CreateProfileMenuItemIndices.Speech)]
     public class MixedRealitySpeechCommandsProfile : ScriptableObject
     {
+        [SerializeField]
+        [Tooltip("Speech System Class to instantiate at runtime.")]
+        [Implements(typeof(IMixedRealitySpeechSystem), TypeGrouping.ByNamespaceFlat)]
+        private SystemType speechSystemType = null;
```

Comment:
I really feel like this should be next to the toggle for enabling and disabling the speech

Reply:
Done